src/v/wasm/wasmtime.cc

```diff
@@ -65,6 +63,8 @@
 #include <unistd.h>
 #include <utility>
 #include <variant>
+#include <wasm.h>
+#include <wasmtime.h>
```

Comment:
quoted include preferred or doesn't work in this case?

Reply:
Ambiguous! This file's header is "wasmtime.h". I will cleanup later after cmake is gone (which requires system includes)